pkg/instancetype/controller/vm/controller_test.go

```diff
@@ -37,6 +37,7 @@ import (
 	virtconfig "kubevirt.io/kubevirt/pkg/virt-config"
 	"kubevirt.io/kubevirt/pkg/virt-config/featuregate"
 	"kubevirt.io/kubevirt/pkg/virt-controller/watch/common"
+	"kubevirt.io/kubevirt/tests/framework/matcher"
```

Comment:
We should avoid importing test code in the production code base. Although it would be convenient...

Reply:
What are you classing as production code here? The `vm_test` package or just the overall path?

Reply:
Overall path